README.md

```diff
@@ -278,7 +282,7 @@ yarn test_functional
 This can be limited to a single spec using:
 
 ``` shell
-yarn test_functional --spec ./tests/functional/spec/exit.spec.js
+make test-functional-spec SPEC=<spec>
```

Comment:
There is a yarn command above this which should be replaced.
There are multiple others which should be replaced with the `make` counterparts.